JudoPayDotNet/Models/CheckCardModel.cs

```diff
@@ -124,6 +125,67 @@ public CheckCardModel()
         /// </value>
```

Comment:
As CheckCardModel and RegisterCardModel have the same attributes it may be better to have one extend the other (with no new attributes), to reduce the chance that changes in future are added to one but not the other